permissions/Base.permission

```diff
@@ -26,7 +26,7 @@ include disable-common.inc
 # include disable-devel.inc
 # include disable-exec.inc
 # include disable-interpreters.inc
-include disable-passwdmgr.inc
+# include disable-passwdmgr.inc
```

Comment:
If the file doesn't exist anymore on the new version, just remove the whole reference to it.